player-item created

zeroadh opened the pull request:
created player-item component and added  player-item tag to round-page.js

Let me know if I missed anything.

zeroadh committed:
player-item created

created player-item component and added to  player-item tag to round-page.js

tylergraf commented on src/components/player-item.js:
```diff
@@ -0,0 +1,27 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class PlayerItem extends LitElement {
+  constructor(){
```
We don't need this constructor here.

tylergraf reviewed (changes requested):
Will you also include this component in `players-crud.js`?

zeroadh committed:
player-item && player-crud

removed constructor in player-item and added player-item constructor to player-crud
